content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -112,26 +112,31 @@ Incoming transactions with unclear purposes may be an inconvenience for financia
 
-- `asfDisallowIncomingCheck`
-- `asfDisallowIncomingNFTOffer`
-- `asfDisallowIncomingPayChan`
-- `asfDisallowIncomingTrustline`
+If the [DisallowIncoming amendment][] :not_enabled: is enabled, you also have the option to block all incoming Checks, NFTokenOffers, Payment Channels, and trust lines. It is generally harmless to be on the receiving end of these objects, but they can block you from deleting your account and it can be confusing to have objects you didn't expect mixed in with the list of objects you created. To block incoming objects, use one or more of these account flags:
+
+- `asfDisallowIncomingCheck` - for Check objects
+- `asfDisallowIncomingNFTOffer` - for NFTokenOffer objects
+- `asfDisallowIncomingPayChan` - for PayChannel objects
+- `asfDisallowIncomingTrustline` - for RippleState (trust line) objects
+
+When a transaction would create one of these ledger entries, if the destination account has the corresponding flag enabled, the transaction fails with the result code `tecNO_PERMISSION`. Unlike Deposit Authorization, these settings do not prevent you from receiving payments in general. Also, enabling this setting doesn't stop you from creating these types of objects yourself (unless the destination of your transaction is also using the setting, of course).
 
-If a destination account has these flags enabled, `rippled` won't create the corresponding object on the ledger and returns the error code `tecNO_PERMISSION`.
 
 ## TransferRate
 
-The `TransferRate` field specifies a fee to charge whenever counterparties transfer the currency you issue. See [Transfer Fees](transfer-fees.html) for more information.
+The `TransferRate` field specifies a fee to charge whenever counterparties transfer the currency you issue.
+
+In the HTTP and WebSocket APIs, the transfer fee is represented as an integer, the amount that must be sent for 1 billion units to arrive. For example, a 20% transfer fee is represented as the value `1200000000`.  The value cannot be less than 1000000000. (Less than that would indicate giving away money for sending transactions, which is exploitable.) You can specify `0` as a shortcut for `1000000000`, meaning no fee.
 
-In `rippled`'s WebSocket and JSON-RPC APIs, the transfer fee is represented as an integer, the amount that must be sent for 1 billion units to arrive. For example, a 20% transfer fee is represented as the value `1200000000`.  The value cannot be less than 1000000000. (Less than that would indicate giving away money for sending transactions, which is exploitable.) You can specify 0 as a shortcut for 1000000000, meaning no fee.
+See [Transfer Fees](transfer-fees.html) for more information.
 
 ## NFTokenMinter
 
-To remove an authorized minter, set `ClearFlag` to 10 (`asfAuthorizedNFTokenMinter`).
+To remove an authorized minter, set `ClearFlag` to 10 (`asfAuthorizedNFTokenMinter`) and omit the `NFTokenMinter` field.
```

Comment:
I noticed this sentence kind of glossed over an important detail here, so I fixed it.